launchdarkly-server-sdk/src/client.rs

```diff
@@ -1632,6 +1655,431 @@ mod tests {
         assert_eq!(evaluated_stage, stage);
     }
 
+    #[test_case(Stage::Off, Operation::Read, vec![Origin::Old])]
```

Comment:
It would be nice if we could move more tests to being "integration" tests so we could get them out of this file. Not something for this PR, but just a general sentiment of I wish less stuff was in this file.